expert-forecast/update-expert.R

```diff
@@ -0,0 +1,25 @@
+# Download and format expert elicitation forecasts for given forecast date
+library(tidyverse); library(googlesheets4); library(SHELF); library(RColorBrewer)
+
+# Set up functions
+source("expert-forecast/get-expert.R")
+source("expert-forecast/format-expert.R")
+source("expert-forecast/plot-expert.R")
+
+
+# Define forecast submission date (a Monday)
+def_date <- "2020-06-29"
```

Comment:
should this be hardcoded? default to a date and take a command line argument or be set externally and then set internally if NULL or.... lots of options. Current set up means this can't be run from another script without editing this one